### PR TITLE
Add Grafana how-to guide

### PR DESCRIPTION
**Pull request by matkuliak**
There was only a blogpost about this, should be a good addition to how-tos

## Summary of the changes / Why this is an improvement

Grafana setup added to how-tos

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/visualize-data-with-grafana.rst [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+   :local:
+
+
+.. _grafana-load-dataset:
+
+Load a sample dataset
+=====================
+
+To visualize data with Grafana, a dataset is needed first. In this sample, demo
+data is added directly via the CrateDB Cloud Console. To import the data go to the Overview page of your deployed cluster.
```
Comment (matthijskrul) [inline docs/visualize-data-with-grafana.rst:29]: Please wrap conforming to style guide

File: docs/visualize-data-with-grafana.rst [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+To visualize data with Grafana, a dataset is needed first. In this sample, demo
+data is added directly via the CrateDB Cloud Console. To import the data go to the Overview page of your deployed cluster. 
+
+.. image:: _assets/img/grafana-overview.png
+   :alt: Cloud Console cluster overview
+
+Once on the Overview page, click on the *Learn how to import data to your
+cluster* link in the bottom part of the Console. A window with 2 SQL
+statements will appear. The first of them creates a table that will host the data
+from NYC Taxi & Limousine Commission which is used in this example. The second statement imports the data into the table created in the first step. These statements must be executed in order. First "1. Create the table" and then "2. Import the data".
```
Comment (matthijskrul) [inline docs/visualize-data-with-grafana.rst:37]: Please wrap conforming to style guide

**Review by matthijskrul — APPROVED** [review]
Very cool! Just needs some line wrapping

File: docs/visualize-data-with-grafana.rst [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+   :alt: Grafana Settings
+
+Once there, click on the *Add data source* button. Here, look up and choose
+"PostgreSQL".
+
+.. image:: _assets/img/grafana-search.png
+   :alt: Grafana integrations
+
+Once "PostgreSQL" is chosen, you will be brought to a form that you must fill
+out to connect to the CrateDB Cloud. Filled out example might look like the
```
Comment (kojinkai) [inline docs/visualize-data-with-grafana.rst:89]:
```suggestion
A filled out example might look like the
screenshot below
```

or maybe better would be

```suggestion
A completed example might look like the
screenshot below
```

File: docs/visualize-data-with-grafana.rst [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+screenshot below.
+
+.. image:: _assets/img/grafana-connection.png
+   :alt: Grafana connection form
+
+What can be different for you is *host* and *user* credentials. The host can be
```
Comment (kojinkai) [inline docs/visualize-data-with-grafana.rst:95]:
```suggestion
The *host* and *user* credentials may appear differently for you. The host can be
```

File: docs/visualize-data-with-grafana.rst [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+When you're satisfied with the look of the panel, click *Apply*. This will
+bring you back to the overview of the dashboard. Now it will have 1 panel
+created in it. Click on the *Add panel* in the top menu bar and you can create
+another one.
+
+.. image:: _assets/img/grafana-add-panel.png
+   :alt: Grafana add another panel to dashboard
+
+Another question worth asking might be: What was the average distance per ride
+per day? To find that out, input this to the SQL console of the new panel:
```
Comment (kojinkai) [inline docs/visualize-data-with-grafana.rst:186]:
```suggestion
per day? To find this out, input the following SQL command into console of the new panel:
```

**Review by kojinkai — APPROVED** [review]
Looks like a very nice and useful tutorial 👍🏼

File: docs/visualize-data-with-grafana.rst (hunk 1 above)
Comment (matkuliak) [reply docs/visualize-data-with-grafana.rst:29]: Ah, right.. good catch. I was playing with auto-wrapper throughout work on this. Must have left it on soft-wrap.

**Pull request merged**